### PR TITLE
Add Mozilla and espace-saint-martin

### PR DESCRIPTION
**Pull request by dns2utf8**
All our confirmed sponsors:
- [x] Mozilla
- [x] Espace St. Martin
- [x] Parity
- [x] Inria
- [x] Bitfury
- [x] Snips

### CODE REVIEW

File: _sponsors/inria.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+name: "BitFury"
+link: "https://bitfury.com/"
```
Comment (rnestler) [inline _sponsors/inria.md]: This one looks wrong. The filename says inria, but it looks like a copy of bitfury.md

File: _sponsors/bitfury.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+name: "BitFury"
+link: "https://bitfury.com/"
+featured: false
+group: sponsor
```
Comment (rnestler) [inline _sponsors/bitfury.md:5]: `png: false` is missing.

File: _sponsors/parity.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+---
+name: "Parity"
+link: "https://www.parity.io/"
+featured: true
+group: sponsor
```
Comment (rnestler) [inline _sponsors/parity.md:5]: `png: false` seems missing.

**Review by rnestler — CHANGES_REQUESTED** [review]
There is also the inconsistency that some files contain the comments describing the content and others don't.

File: _sponsors/inria.md (hunk 1 above)
Comment (Hoverbear) [reply _sponsors/inria.md]: Yup this one is wrong. :)

**Comment by dns2utf8** [comment]
Thanks @rnestler 
Should we merge this so we can see it in action?

**Comment by badboy** [comment]
@dns2utf8 Did you ensure that it looks good locally?

**Comment by dns2utf8** [comment]
It looks okay locally and it should integrate well with the cleanup PR #38

**Review by rnestler — APPROVED** [review]

**Pull request merged**